pkg/commands/drift/statefiles/statefiles.go

```diff
@@ -86,6 +98,43 @@ func (c *DriftStatefilesCommand) Flags() *cli.FlagSet {
 		DefaultIssueLabel: "guardian-statefile-drift",
 	})
 
+	// Command options
+	f := set.NewSection("COMMAND OPTIONS")
+
+	f.StringVar(&cli.StringVar{
+		Name:    "organization-id",
+		Target:  &c.flagOrganizationID,
+		Example: "123435456456",
+		Usage:   `The Google Cloud organization ID for which to detect drift.`,
+	})
+
+	f.StringVar(&cli.StringVar{
+		Name:    "github-repo-query",
+		Target:  &c.flagGitHubRepoQuery,
+		Example: "labels.terraform:*",
+		Usage:   `The label to use to find GCS buckets with Terraform statefiles.`,
+	})
```

Comment:
Is this used? Seems like logic switch to topics instead of labels.

Reply:
Removed